### PR TITLE
add: Vanilla template

### PR DESCRIPTION
**Pull request by rajarju**
## Description

Enabled the option to add Vanilla HTML/CSS/JS

## Related Issue

Fix: #96 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by purplecabbage — CHANGES_REQUESTED** [review]
I think I was expecting that both react-spectrum and vanilla version would load in experience.adobe.
The option to choose a deploy technology is kinda confusing imho.

File: test/generators/add-web-assets/index.test.js [hunk 1]
```diff
@@ -36,15 +36,15 @@ afterAll(() => {
...
-//     { name: 'Exc Shell React', value: expect.stringContaining(n('exc-react/index.js')) },
-//     { name: 'Raw HTML/JS', value: expect.stringContaining(n('raw/index.js')) }
-//   ],
-//   validate: utils.atLeastOne
-// })]
+const expectedPromptChoices = [expect.objectContaining({
+  type: 'list',
+  name: 'webAssetsGenerator',
+  choices: [
+    { name: 'Adobe Experience Cloud Shell - React', value: expect.stringContaining(n('exc-react/index.js')) },
```
Comment (meryllblanchet) [inline test/generators/add-web-assets/index.test.js:43]: React Spectrum 3 UI

File: test/generators/add-web-assets/index.test.js [hunk 2]
```diff
@@ -36,15 +36,15 @@ afterAll(() => {
...
+    { name: 'Raw HTML/JS', value: expect.stringContaining(n('raw/index.js')) }
```
Comment (meryllblanchet) [inline test/generators/add-web-assets/index.test.js:44]: Raw HTML/JS UI

(Please note that this UI will also be able to deploy within Adobe Experience Cloud, so it should also integrate with the ExC Shell client-side API.

File: generators/add-web-assets/index.js [hunk 3]
```diff
@@ -52,25 +52,30 @@ class AddWebAssets extends Generator {
...
-    this.composeWith(webAssetsGenerator, this.options)
+    if (!this.options['skip-prompt']) {
+      const promptProps = await this.prompt([
+        {
+          type: 'list',
+          name: 'webAssetsGenerator',
+          message: 'Which type of UI do you want to add to your project?\nselect template to generate',
+          choices: [
+            {
+              name: 'Adobe Experience Cloud Shell - React',
```
Comment (meryllblanchet) [inline generators/add-web-assets/index.js:63]: React Spectrum 3 UI

File: generators/add-web-assets/index.js [hunk 4]
```diff
@@ -52,25 +52,30 @@ class AddWebAssets extends Generator {
...
+              value: excReactWebAssetsGenerator
+            },
+            {
+              name: 'Raw HTML/JS',
```
Comment (meryllblanchet) [inline generators/add-web-assets/index.js:67]: Raw HTML/JS UI

(Please note that this UI will also be able to deploy within Adobe Experience Cloud, so it should also integrate with the ExC Shell client-side API).

File: generators/add-web-assets/exc-react/templates/src/index.js [hunk 5]
```diff
@@ -49,9 +49,6 @@ function bootstrapInExcShell () {
   // get the Experience Cloud Runtime object
   const runtime = Runtime()
 
-  // set the app name to the Shell header
-  runtime.customEnvLabel = '<%= projectName %>'
```
Comment (rajarju) [inline generators/add-web-assets/exc-react/templates/src/index.js:53]: @purplecabbage Is this no longer required?

Comment (purplecabbage) [reply generators/add-web-assets/exc-react/templates/src/index.js:53]: It is ugly. Define 'required' ;)

Comment (purplecabbage) [reply generators/add-web-assets/exc-react/templates/src/index.js:53]: The intended use of this is to bring attention to 'beta' releases, or other environmental factors.
> " ... using it as an app title is not recommended."

**Review by purplecabbage — APPROVED** [review]

File: generators/add-web-assets/exc-react/templates/src/index.js (hunk 5 above)
Comment (rajarju) [reply generators/add-web-assets/exc-react/templates/src/index.js:53]: I thought FF apps were required to show this on EXC shell, 
But now I see, it is to show env and not app name

**Pull request merged**